app/src/main/res/drawable/rounded_border.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+       android:shape="rectangle">
+
+    <stroke
+            android:width="4dp"
+            android:color="@color/colorPrimary" />
+    <solid
+    android:color="@color/colorPrimary"/>
+
+    <corners
+            android:topLeftRadius="15dp"
+            android:topRightRadius="15dp"
+            />
+</shape>
```

Comment:
Can't say I'm a fan of the rounded border 👀

Reply:
Yea in retrospect, it's kinda off. Gonna take it out.

Reply:
Fixed here https://github.com/power-meter/android-client/pull/4/commits/3cc87553796d48fa05c96273be6e13b128a22b27